### PR TITLE
DAT-567: Footer pages

### PR DESCRIPTION
**Pull request by kjoshi**
This PR addresses [DAT-567](https://london.atlassian.net/browse/DAT-567) and adds content for all of the pages for which we have links in the footer.

All of the content has been copied over from the existing London Datastore. Some of the pages link to other pages on the London Datastore, but Sven has said that's Ok for now.

I've also made some minor changes to the CSS, because the sizes and spacing between the `h1`, `h2` and `h3` elements was making the pages look a bit odd, in my opinion.

### CODE REVIEW

**Comment by madelinekosse** [comment]
Maybe I'm reading the ticket wrong, but is this meant to include the "about" page? I see CKAN marketing copy on there

Edit: never mind - I just had to restart ckan to pick it up. Worth noting if that happens in CI too.

File: ckanext/gla/templates/about/accessibility.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<div class="module-content">
+    <h1>Accessibility</h1>
+    <p>This website is run by the Greater London Authority (GLA). We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
+    <ul>
+    <li>change colours, contrast levels and fonts</li>
+    <li>zoom in up to 300% without the text spilling off the screen</li>
+    <li>navigate most of the website using just a keyboard</li>
+    <li>navigate most of the website using speech recognition software</li>
+    <li>listen to most of the website using a screen reader (including the most</li>
+    <li>recent versions of JAWS, NVDA and VoiceOver)</li>
```
Comment (madelinekosse) [inline ckanext/gla/templates/about/accessibility.html]: I don't think this should be a new `<li>` - it's one sentence:
image

File: ckanext/gla/templates/about/frequently-asked-questions.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <h2>What data cannot or will not be released?</h2>
+    <p>Our policy is to only withhold data where to release it would infringe privacy legislation or a contractual obligation eg commercial confidentiality.</p>
+    <h2>Isn’t some of this data available elsewhere?</h2>
+    <p>We recognise that some organisations already provide open access to their data and where appropriate we have provided a direct link to their website, all available from one place.&nbsp; However we feel that there are some instances where we can add value to this, for example by focussing on just the data for London, by saving you from having to download separate datasets for multiple years, or by making it available in a standardised and easy to use format.</p>
+    <h2>Will the Datastore duplicate information held in London.gov.uk website?</h2>
+    <p>Whilst this is possible, the idea behind the Datastore is to provide raw data that the GLA holds, leaving the main website of the GLA, www.london.gov.uk to make available information in its final state (eg. Policy and Strategy documents).</p>
```
Comment (madelinekosse) [inline ckanext/gla/templates/about/frequently-asked-questions.html]: Should this be an href? Github didn't highlight the line properly, but the `www.london.gov.uk` part above

File: ckanext/gla/templates/about/frequently-asked-questions.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <p>The Office for National Statistics website contains a summary of the terms of use associated with Crown Copyright material from ONS. It explains the terms for personal and research use, plus the need for a click-use licence from OPSI for commercial or other uses. More information is available at <a href="http://www.ons.gov.uk/copyright/index.html">http://www.ons.gov.uk/copyright/index.html</a></p>
+    <p>We hope to be able to provide more information about the licensing terms for GLA created data in due course.&nbsp; We are seeking advice from the <a href="http://www.okfn.org/">Open Knowledge Foundation</a> about the best way of doing this.</p>
+    <h2>Who do I contact if I have a comment or complaint about this site?</h2>
+    <p>The GLA welcomes comments about this site. In the first instance, email us at datastore@london.gov.uk. This e-mail will be received by the Datastore Project Team. We will try to respond as soon as possible. If you have a general enquiry, comment or complaint about the GLA, please follow the link below, which will give you the appropriate contact details: http://www.london.gov.uk/gla/contact.jsp</p>
```
Comment (madelinekosse) [inline ckanext/gla/templates/about/frequently-asked-questions.html]: Should also be an href on the link?

**Review by madelinekosse — APPROVED** [review]
If your job was just to copy stuff from elsewhere into these pages, then I approve the changes. Left some comments where I think the content's not quite right but GIGO from a dev perspective

**Comment by kjoshi** [comment]
Thanks, I've fixed those things

**Pull request merged**